helix-core/src/main/java/org/apache/helix/controller/pipeline/AbstractBaseStage.java

```diff
@@ -64,9 +65,9 @@ public String getStageName() {
     return className;
   }
 
-  public static <T> Future asyncExecute(ExecutorService service, Callable<T> task) {
+  public static <T> Future<T> asyncExecute(ExecutorService service, Callable<T> task) {
```

Comment:
Why to change this?

Reply:
Ok, I now understood what you mean. I applied the reformatting to all files I touched within the change

Reply:
Let's minimize the this kind of change. The reason is that Helix release not only support java 11 but backward compatible with 8.

Any this kind of "improvement" for working code may break backward build.

Reply:
This "improvement" would have been compatible with Java 1.5, for all I know, but OK, your choice. Rolled back.